legal-api/report-templates/template-parts/business-summary/foreignJurisdiction.html

```diff
@@ -0,0 +1,64 @@
+<div class="no-page-break">
+    <div class="separator mt-4"></div>
+    <div class="section-title mt-4">Previous Foreign Jurisdiction Information</div>
+    <table class="section-data nr-name-table mt-4" role="presentation">
+        <tr>
+            <td class="col-25">
+                <div class="section-sub-title">Name in Foreign Jurisdiction</div>
+                <div class="pt-2">
+                    {{continuationIn.foreignJurisdiction.legalName}}
+                </div>
+            </td>
+            <td class="col-25">
+                <div class="section-sub-title">Identifying Number in Foreign Jurisdiction</div>
+                <div class="pt-2">
+                    {{continuationIn.foreignJurisdiction.identifier}}
+                </div>
+            </td>
+        </tr>
+        <td colspan="2" style="height: 20px;"></td>
+        <tr>
+            <td class="col-25">
+                <div class="section-sub-title">Date of Incorporation, Continuation or <br> Amalgamation in Foreign Jurisdiction</div>
+                <div class="pt-2">
+                    {{continuationIn.foreignJurisdiction.incorporationDate}}
+                </div>
+            </td>
+            <td class="col-25">
+                <div class="section-sub-title">Foreign Jurisdiction</div>
+                <div class="pt-2">
+                    {% if continuationIn.foreignJurisdiction.region %}
+                        {{continuationIn.foreignJurisdiction.region}}
+                    {% else %}
+                        {{continuationIn.foreignJurisdiction.country}}
+                    {% endif %}
+                </div>
+            </td>
+        </tr>
+    </table>
+</div>
+
+{% if continuationIn.mode == 'EXPRO' %}
```

Comment:
unsure if this is what is intended for expro that is to be made historical

Reply:
as mentioned we cannot depend on mode